RailDriver.txt

```diff
@@ -669,6 +669,20 @@ elseif (RailDriverState == 2)
                 semControlTrucks(Trucks, Throttle, Brake, Reverser)
             ]#
 
+            # Set the brakes, if the train is not moving, the setpoint is 0, and the Emergency Brake is not active.
+            # This is to prevent the train from rolling away. Fixes #65.
+            if (SpeedAbs == 0 & pidGetSetPoint(PIDF) == 0 & EmBrakeState == 0)
```

Comment:
`EmBrakeState` condition here _may_ be interfering with the Emergency Brake.

When the Emergency Brake is tested, a Rollback Runaway is observed shortly after the train has stopped when going uphill. This needs further investigation.

Reply:
As I initially thought, the  `EmBrakeState` was interfering with the Emergency Brake.
I have a staged commit that fixes this interference.